src/main/webapp/app/modules/login/LoginForm.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { TextField, Flex, Form, Checkbox, Button, Text } from '@adobe/react-spectrum';
+import Typography from '@material-ui/core/Typography';
+
+export interface ILoginProps {
+  loginError: boolean;
+  handleLogin: Function;
```

Comment:
```suggestion
  handleLogin:(input: string) => bool;
```
Can we type the functions signature please?

Reply:
props are coming from parent component...type of the prop is defined whether it is function/array/variable

Reply:
What do you mean by `type of the prop is defined whether it is function/array/variable` ? I understand props passed in from the parent but, why does that affect the above mentioned type change?

Reply:
may i did not get the your point..i just checked the typescript doc...updated the function type in interface as below..

(username: string,password: string, rememberMe: boolean) => void;